lib/shopify_transporter/pipeline/magento/product/top_level_variant_attributes.rb

```diff
@@ -8,6 +8,7 @@ module Magento
       module Product
         class TopLevelVariantAttributes < Pipeline::Stage
           def convert(hash, record)
+            return if record['variants'].nil?
```

Comment:
why is this needed now but wasn't before?

Reply:
Probably fails if anything called `simple_product_in_magento_format` as that assumes the presence of `variants`.

Reply:
This line of code should've been added when I was working on the ticket about top level variant fields. Converting top level variants makes no sense if there're no variants in the first place. I added this line just for safety.